docs/GETTING_STARTED.md

```diff
@@ -185,13 +185,13 @@ To further convert ONNX model to IR you can run the script:
 In order to test the model being exported via a `tools/export.py` script run the following command:
 
 ```shell
-# python tools/test_exported.py config.py ${DEPLOY_DIR}/checkpoint.xml --out /tmp/out.pkl --backend openvino --do_not_normalize
```

Comment:
please fix it in other *.md files

Reply:
Which ones?

Reply:
in OTE, non-existing PR that you should create :)

Reply:
Ok then :smile: